### PR TITLE
Python test

### PR DESCRIPTION
**Pull request by MHOAKEY**
Not yet in form of a function

### CODE REVIEW

File: test2.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# away it is until that date.
+
+# Example:
+# input: 2022-06-22
+# output: 1 day ago (this is based on the function finding that today's date is
+# 2022-06-23 this will obviously vary based on what day it currently is)
+
+
+def timeBetweenNowAndThen(dateInput):
+    today = datetime.datetime.now()
```
Comment (oakeef) [inline test2.py:15]: do you need datetime.datetime.now() ? or is this an accidental repetition?

File: test2.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        int(userYear), int(userMonth), int(userDay))
+    userUnixTime = (time.mktime(userYearMonthDay.timetuple()) * 1000)
+    unixBetweenDates = todayUnix - userUnixTime
+    unixYearsBetween = unixBetweenDates / 31_536_000_000
+    unixMonthsBetween = (unixBetweenDates -
+                         (int(unixYearsBetween) * 31_536_000_000)) / 2_629_800_000
+    unixDaysBetween = ((unixBetweenDates - (int(unixYearsBetween) * 31_536_000_000)
+                        ) - (int(unixMonthsBetween) * 2_629_800_000)) / 86_400_000
+
+    result = str(int(unixYearsBetween)), " Year(s), ", str(int(
```
Comment (oakeef) [inline test2.py]: if you replace the `,` with `+` here it will make result a string and you won't need lines 33-35

File: test2.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+# Output: [11,13,17,19]
+
+
+num1 = int(input("Enter number: "))
+num2 = int(input("Enter larger number: "))
+
+
+def primeNumbersInRange(a, b):
+    array = []
+    x = range(a, b + 1)
```
Comment (oakeef) [inline test2.py]: you could probably move this down onto the line below instead of using a variable

Comment (MHOAKEY) [reply test2.py]: makes sense! I have that done now. Thank you

**Pull request merged**